build.sbt

```diff
@@ -1046,6 +1046,9 @@ lazy val `scio-parquet` = project
       "org.slf4j" % "slf4j-api" % slf4jVersion,
       // provided
       "org.tensorflow" % "tensorflow-core-api" % tensorFlowVersion % Provided,
+      // runtime
+      "org.apache.hadoop" % "hadoop-client" % hadoopVersion % Runtime excludeAll (Exclude.metricsCore),
```

Comment:
This dep is declared as provided in beam parquet io [here](https://github.com/apache/beam/blob/e6e180b68965e872822f5f32281b30c65da16af3/sdks/java/io/parquet/build.gradle#L51C3-L51C11).

Moved the dependency from `scio-smb` to `scio-parquet`